src/Ztm.Configuration/BitcoinAddressConfigurationConverter.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.ComponentModel;
+using System.Text.RegularExpressions;
+using NBitcoin;
+using Ztm.Zcoin.NBitcoin;
+
+namespace Ztm.Configuration
+{
+    public sealed class BitcoinAddressConfigurationConveter : TypeConverter
+    {
+        NetworkType GetNetworkType(string networkName)
+        {
+            NetworkType networkType;
+            if (Enum.TryParse<NetworkType>(networkName, out networkType))
+            {
+                return networkType;
+            }
+
+            throw new ArgumentException($"Value is not valid.", nameof(networkType));
+        }
+
+        (string, string) ParseAddress(string address)
+        {
+            Regex rx = new Regex(@"^(\w+):(\w+)$", RegexOptions.Compiled);
```

Comment:
We should make this one as static so it will not compile every time.

Reply:
make sense